lesson05/script.js

```diff
@@ -0,0 +1,41 @@
+let input;
+let availableBalance = 1000;
+let withdrawAmount = 0;
```

Comment:
withdrawalAmount and depositAmount don't need to be declared here, you could move them inside of your loop and just declare them on the same line as the prompt.